hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/SCMDeletedBlockTransactionStatusManager.java

```diff
@@ -524,7 +527,12 @@ public void commitTransactions(
     }
     try {
       deletedBlockLogStateManager.removeTransactionsFromDB(txIDsToBeDeleted);
-      metrics.incrBlockDeletionTransactionCompleted(txIDsToBeDeleted.size());
+      int numTransactionsCompleted = txIDsToBeDeleted.size();
+      metrics.incrBlockDeletionTransactionCompleted(numTransactionsCompleted);
```

Comment:
These metrics are not required, total can be derived from sucess+failure, and last iteration additional metrics not required.